keyboards/keychron/c1/keyboard.json

```diff
@@ -0,0 +1,259 @@
+{
+    "manufacturer": "Keychron",
+    "keyboard_name": "Keychron C1",
+    "maintainer": "NetUserGet",
+    "bootloader": "sn32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+        "dip_switch": true
+    },
+    "lto": true,
+    "indicators": {
+        "caps_lock": "B10"
+    },
+    "dip_switch": {
+        "pins": ["D4"]
+    },
+    "matrix_pins": {
+        "cols": ["A8", "A9", "A10", "A11", "A12", "A13", "A14", "A15", "B0", "B1", "B2", "B3", "B4", "B5", "B6", "B7", "B8"],
+        "rows": ["C15", "D11", "D10", "D9", "D8", "D7"]
+    },
+    "debounce": 0,
```

Comment:
don't you have chatter issues with debounce disabled?

Reply:
Nope.